src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { Box, Button, Typography } from '@mui/material'
+
+const Dashboard = () => {
+  const routesNotAuthed = [{ path: '/', label: 'Register/Login page' }]
+  const routesAuthed = [
+    { path: '/callback', label: 'Auth Callback' },
+    { path: '/fill-profile', label: 'Fill Profile' },
+    { path: '/friends', label: 'Friends' },
+    { path: '/messages', label: 'Messages' },
+    { path: '/who-liked-you', label: 'Your Likes List' },
+    { path: '/near-me', label: 'Near Me' },
+    { path: '/my-account', label: 'My Account' },
+  ]
+
+  const routesTemporaryExamples = [
+    { path: '/report', label: 'Report Dialog Example' },
+    { path: '/delete', label: 'Delete User Dialog Example' },
+    { path: '/invite', label: 'Invitation' },
+    { path: '/error-400', label: 'Error 400' },
+    { path: '/error-500', label: 'Error 500' },
+    { path: '*', label: 'Error Page (Catch-All)' },
+    { path: '/new-match', label: 'New Match' },
+  ]
+
+  const routesTemporary = [
+    { path: '/account', label: 'User Account' },
```

Comment:
I propose to add some information for page account: in order for the page to work, you need to log in, have a profile saved in DB and launch the backend.